src/frontend/src/env/schema/env-token-ckerc20.schema.ts

```diff
@@ -13,7 +13,7 @@ const EnvCkErc20TokenDataSchema = EnvIcTokenSchema.extend({
 });
 
 export const EnvCkErc20TokensSchema = z.record(
-	envTokenSymbol,
+	EnvTokenSymbolSchema,
 	z.union([z.undefined(), EnvCkErc20TokenDataSchema])
```

Comment:
you don't want to move this one in this same PR, right?

Reply:
That's another task, which I wasn't planing to solve today. We need a generic `z.union([z.undefined(), T])` because this union is used in #3506 as well.